helloworld-singleton/src/test/java/org/jboss/as/quickstarts/singleton/BasicRuntimeIT.java

```diff
@@ -54,6 +54,6 @@ public void testHTTPEndpointIsAvailable() throws IOException, InterruptedExcepti
         final HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
         Assertions.assertEquals(200, response.statusCode());
         final String[] bodyLines = response.body().toString().split(System.lineSeparator());
-        Assertions.assertEquals("<meta http-equiv=\"Refresh\" content=\"0; URL=home.jsf\">", bodyLines[bodyLines.length-3]);
+        Assertions.assertEquals("</div>", bodyLines[bodyLines.length-3]);
```

Comment:
I'd recommend asserting line 23 instead of line 33.

Reply:
made the required changes. Thanks.